.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -9,7 +9,9 @@ body:
       description: |
         This should include as little code as possible, and no extraneous dependencies. Do not share your entire project.
         You should also test your issue against the latest NPM package version(s) to ensure that it hasn't already been resolved.
-        Sharing a link to a GitHub repository, [Codesandbox](https://codesandbox.io) and/or preview deployment are best. Include any instructions to run the project in the README or the summary field below.
+        Sharing a link to a GitHub repository or preview deployment are best. Include any instructions to run the project in the README or the summary field below.
+
+        Don't have an available repo/preview to share? You can create online with our [StackBlitz template](https://stackblitz.com/edit/web3modal-next-template) quickly.
```

Comment:
@boidushya I think we could replace this with a template which created by the company account.

Reply:
I feel we can merge this but it would definetly be better to have it owned by WC directly or mb dev-rel team at least